src/main/webapp/app/exercises/programming/assess/programming-assessment-dashboard/programming-assessment-dashboard.component.html

```diff
@@ -30,12 +30,17 @@ <h4 [translateValues]="{ exerciseTitle: exercise.title }" jhiTranslate="artemisA
                         <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessor">Reviewer</a>
                         <fa-icon [icon]="'sort'"></fa-icon>
                     </th>
-                    <th *ngIf="newManualResultAllowed">
-                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentFirst">Assessment</a>
-                    </th>
-                    <th *ngIf="numberOfCorrectionrounds == 2 && newManualResultAllowed">
-                        <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentSecond">Assessment</a>
-                    </th>
+                    <ng-container *ngIf="newManualResultAllowed">
+                        <th *ngIf="exercise.course || (exercise.exerciseGroup && numberOfCorrectionrounds == 1)">
+                            <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.action">Assessment</a>
+                        </th>
+                        <th *ngIf="numberOfCorrectionrounds == 2">
+                            <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentFirst">Assessment</a>
+                        </th>
```

Comment:
is this a mistake and you meant to have `== 1`?
```suggestion
                        <th *ngIf="numberOfCorrectionrounds == 1">
                            <a class="th-link" jhiTranslate="artemisApp.assessment.dashboard.columns.assessmentFirst">Assessment</a>
                        </th>
```

Reply:
No, here i need to check if there numberOfCorrectionRounds is 2. (If there are 2, I want to display the column title for the first correction, and for the second correction)